Mathlib/Data/Finsupp/MonomialOrder.lean

```diff
@@ -152,4 +153,23 @@ theorem MonomialOrder.lex_le_iff [WellFoundedGT σ] {c d : σ →₀ ℕ} :
 theorem MonomialOrder.lex_lt_iff [WellFoundedGT σ] {c d : σ →₀ ℕ} :
     c ≺[lex] d ↔ toLex c < toLex d := Iff.rfl
 
+theorem MonomialOrder.lex_unit_lt_iff {c d : Unit →₀ ℕ} :
```

Comment:
Should that be simp? Should be stated for all `Unique` types instead of merely `Unit`?

Reply:
Adjusted